src/components/Send/ShowUtxos.tsx

```diff
@@ -295,22 +295,9 @@ const ShowUtxos = ({
   const [showFrozenUtxos, setShowFrozenUtxos] = useState<boolean>(false)
 
   // Handler to toggle UTXO selection
-  const handleUtxoCheckedState = useCallback(
-    (utxoIndex: number, isFrozen: boolean) => {
-      if (!isFrozen) {
-        const utxos = unFrozenUtxos.map((utxo: Utxo, i: number) =>
-          i === utxoIndex ? { ...utxo, checked: !utxo.checked } : utxo,
-        )
-        setUnFrozenUtxos(utxos)
-      } else {
-        const utxos = frozenUtxos.map((utxo: Utxo, i: number) =>
-          i === utxoIndex ? { ...utxo, checked: !utxo.checked } : utxo,
-        )
-        setFrozenUtxos(utxos)
-      }
-    },
-    [frozenUtxos, unFrozenUtxos, setUnFrozenUtxos, setFrozenUtxos],
-  )
```

Comment:
I think shortening this piece of code would give the same results. Not sure though, please go through this.